src/types/seat/seat.rs

```diff
@@ -549,8 +548,8 @@ impl Seat {
     /// Send the modifier state to focused keyboard resources.
     ///
     /// Compositors should use `Seat::keyboard_notify_modifiers()` to respect any keyboard grabs.
-    pub fn keyboard_send_modifiers(&self, modifiers: &mut KeyboardModifiers) {
-        unsafe { wlr_seat_keyboard_send_modifiers(self.data.0, modifiers) }
+    pub fn keyboard_send_modifiers(&self, modifiers: &mut keyboard::KeyboardModifiers) {
+        unsafe { wlr_seat_keyboard_send_modifiers(self.data.0, &mut (*modifiers).into()) }
```

Comment:
Can you store the converted value in a variable for all these functions?

I'm a little unsure how the temporaries would work especially because these are pointers not references (since we are calling C functions), so I'd rather just avoid the ambiguity and make it clear that we need to push the converted value to the stack before passing it as a pointer to these functions.